Project.toml

```diff
@@ -1,22 +1,26 @@
 name = "Quadmath"
 uuid = "be4d8f0f-7fa4-5f49-b795-2f01399ab2dd"
-version = "0.5.13"
+version = "0.5.14"
```

Comment:
I'm not a huge fan of bumping only the patch if we are going to drop support for some old Julia versions. Because ideally we'd keep a `release-0.5` branch going (that keeps support for Julia 1.6 through 1.9), and only drop Julia support on a new release branch.

Maybe we can take this opportunity to release a version 1.0.0 of this package (and 1.0.0 can drop support for Julia 1.6 through 1.9)?